website/docs/src/client-js/internal-links.js

```diff
@@ -32,8 +32,12 @@ export class InternalLinksHandler {
           console.debug('navigateInternal URL parse error', elem, elem.getAttribute('href'), error);
```

Comment:
I know you haven't touched this line but maybe we don't need it there anymore to clean a bit around?

Reply:
This is intentional in case the href could not get read, so we get some output if someone says link xy does not work. Its inside the error block of try/catch and should't get called anyways.